### PR TITLE
first commit! adding myself to the list of birds

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
## Cute animal pic
image

## Link to Trello card (if applicable)
https://trello.com/c/f3rJNpTK/139-welcome-olu

## Description
_The commit messages say what you did; this should explain why and/or how._
- [X] Description is in Trello card

## Steps to test/reproduce
N/A

## Show me
N/A

### CODE REVIEW

**Comment by patriciarealini** [comment]
[Welcome Olu!](https://trello.com/c/f3rJNpTK/139-welcome-olu)

File: content/birds/olu.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+title: Olu Niyi-Awosusi
```
Comment (mirisuzanne) [inline content/birds/olu.md:1]: From the preview, it looks like we need to prefix the yaml data:

```suggestion
---
title: Olu Niyi-Awosusi
```

File: content/birds/olu.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+sub: Javascript Developer
+bird: olu
+date: 2021-06-07
+image:
+  src: [`birds/bird.jpg`]
```
Comment (mirisuzanne) [inline content/birds/olu.md:6]: For the page headers, we've been adding actual photos. You can add that in `src/images/birds/` and preview the page at `/authors/olu/` to make sure you like how it lays out. If needed, you can add a `position` attribute to the `image` data, which sets [object-position](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position) on the image.

**Review by mirisuzanne — COMMENTED** [review]
still working on the sketch for you! :)

**Comment by patriciarealini** [comment]
[Welcome Olu!](https://trello.com/c/f3rJNpTK/139-welcome-olu)

**Comment by stacyk** [comment]
@dvdherron I think you could if you want. Yours could look like a frame was intentionally applied, although the second image isn't centered. I think you could get away with jpg still even if you wanted to crop it a bit so the file sizes would stay small. 
dark-theme-oddsite

File: content/birds/olu.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: Olu Niyi-Awosusi
+sub: Javascript Developer
+bird: olu
+date: 2021-06-07
+# image:
+#   src: [`birds/bird.jpg`]
```
Comment (stacyk) [inline content/birds/olu.md:7]: @oluoluoxenfree if you would prefer to not have a header image, then we probably will want to remove this commented out code. We can add a hero image at any time in the future if you have one you want to use down the road.

**Review by stacyk — APPROVED** [review]
I left a small cleanup comment, but I am going to approve this since I will be out next week.

File: content/birds/olu.md (hunk 3 above)
Comment (mirisuzanne) [reply content/birds/olu.md:7]: (I did the commenting-out, just to keep this from throwing an error. I think we want to have a header image if we can)

Comment (oluoluoxenfree) [reply content/birds/olu.md:7]: @mirisuzanne do i need to delete this or add a header? or was it just a heads up?

Comment (mirisuzanne) [reply content/birds/olu.md:7]: @oluoluoxenfree I think ideally we would add a header image here, like on the other bird pages. Do you have a photo we could use for that?

Comment (oluoluoxenfree) [reply content/birds/olu.md:7]: @mirisuzanne nothing immediately comes to mind 😞 is it okay to just leave it blank?

**Review by jgerigmeyer — APPROVED** [review]

File: content/birds/olu.md (hunk 3 above)
Comment (mirisuzanne) [reply content/birds/olu.md]: @oluoluoxenfree I think we can make it work. I tried adding your sketch as a header image, just so it's a bit more similar to the other bird pages. Does that look ok to you?

**Pull request merged**